### ISSUE TITLE
model.train  project name ,  Do you consider supporting Chinese

### ISSUE THREAD

**Issue by monkeycc** [ultralytics/ultralytics#6268]
```python
from ultralytics import YOLO

# Load a model
model = YOLO('path/to/last.pt')  # load a partially trained model

# Resume training
results = model.train(data="coco128.yaml",project="中文项目",name="检测项目1")
```

```

  File "C:\Users\mm\.conda\envs\Yolov8\Lib\site-packages\torch\serialization.py", line 288, in __init__
    super().__init__(torch._C.PyTorchFileWriter(str(name)))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Parent directory 中文项目\检测项目1\weights does not exist.
```

**Comment by glenn-jocher** [comment]
@monkeycc the error message you're encountering indicates that the parent directory for the project name with Chinese characters does not exist. This suggests a possible issue with the handling of non-ASCII characters in the directory paths.

You can try creating the required directory structure manually before running the training command to ensure that the directories are recognized by your file system. Additionally, ensure that your environment is set up to handle Unicode characters in file paths, as this can sometimes be a source of issues.

If the problem persists, consider using ASCII-only project and name identifiers to avoid potential encoding complications. Adhering to ASCII can help ensure compatibility across different operating systems and environments.

For further assistance or to report a potential bug with handling non-ASCII paths, you may open an issue on the GitHub repo to have the team look into improving this aspect of the software. For guidelines and tips on how to train using custom datasets or projects, you can refer to the documentation at https://docs.ultralytics.com.

**Comment by glenn-jocher** [comment]
@Laughing-q I've added a TODO here to see if we can allow Chinese characters for `project` and `name` arguments. Can you take a look and see if this is possible?

**Comment by Laughing-q** [comment]
@monkeycc @glenn-jocher I think we do support Chinese project name.
```python
from ultralytics import YOLO

# Load a model
model = YOLO("yolov8n.pt")  # load a partially trained model

# Resume training
results = model.train(data="coco128.yaml", project="中文项目", name="检测项目1")
```
above code creates me a Chinese folder of `中文项目/检测项目1`
![pic-231113-0958-41](https://github.com/ultralytics/ultralytics/assets/61612323/042876fc-7cbd-43ed-883c-24ca4625a0c2)

**Comment by Laughing-q** [comment]
@monkeycc It appears that either you gave a wrong location of weight or it's a windows system specific issue， but I dont have a windows environment to verify it.😂

**Comment by glenn-jocher** [comment]
@Laughing-q hmm ok I have a Windows computer at the office, I'll try it out.

**Comment by glenn-jocher** [comment]
@Laughing-q @monkeycc yes, can confirm this crashes on Windows. I'll keep the TODO tag on the issue while I try to find a fix.

**Comment by glenn-jocher** [comment]
@monkeycc @Laughing-q ok I think I've resolved this now in https://github.com/ultralytics/ultralytics/pull/6407/commits/97c876d43b8b172b3a45031df55d9201d525cb30 which adds chinese character support in Windows and even better also brings emoji support:

<img width="1006" alt="Screenshot 2023-11-18 at 03 34 31" src="https://github.com/ultralytics/ultralytics/assets/26833433/9bbe9526-3282-436a-9d2a-15781647c18a">

**Comment by glenn-jocher** [comment]
@monkeycc @Laughing-q good news 😃! Your original issue may now be fixed ✅ in PR #6407 and published in `ultralytics 8.0.212` soon.

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://bit.ly/yolov5-paperspace-notebook"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
